python/ray/tune/experiment/trial.py

```diff
@@ -417,6 +398,41 @@ def __init__(
         self._state_json = None
         self._state_valid = False
 
+    def create_placement_group_factory(self):
```

Comment:
I'm wondering if we should call this implicitly when `Trial.placement_group_factory`. But also ok to keep this for now

Reply:
good idea. added a TODO for now.
if there are tests failing because of this, I will turn it into a getter.

Reply:
I think we should do this in any case (turn into getter and create PGF on first call). Otherwise it increases the complexity of the Trial class